runtime.md

```diff
@@ -15,3 +15,46 @@ Runs a process in a container. Can be invoked several times.
 Not sure we need that from oc cli. Process is killed from the outside.
 
 This event needs to be captured by oc to run onstop event handlers.
+
+## Hooks
+Hooks allow one to run code before/after various lifecycle events of the container. The state of the container is passed to the hooks over stdin, so the hooks could get the information they need to do their work.
+
+The path to the hooks could be absolute or just the name of the file. Hooks are executed from the host's filesystem. If the path to a hook isn't absolute then it is searched for within the PATH environment variable on the host.
+
+The working directory for the hooks is set to the container's root filesystem unless overriden in the configuration.
```

Comment:
Is this [“the bundle directory”][1] (e.g. `/path/to/bundle`) or [“the container's root filesystem”][2] (e.g. `/path/to/bundle/rootfs`)?  I'd guess it's the latter, but to make things very clear I'd suggest something like:

    … the container's [root filesystem][rootfs] unless overriden in the configuration.

    [rootfs]: config.md#root-configuration

[1]: https://github.com/opencontainers/specs/blob/5c98addb2b07a600c57e3f7229041103abb5c4af/bundle.md
[2]: https://github.com/opencontainers/specs/blob/5c98addb2b07a600c57e3f7229041103abb5c4af/config.md#root-configuration